### PR TITLE
"#1081 Removed public class modifiers"

### PR DESCRIPTION
**Pull request by c-thornton**
Removed public class modifiers from the Meetup.java and MeetupSchedule.java src and .meta folders.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/meetup/.meta/src/reference/java/Meetup.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import java.time.DayOfWeek;
 
 
-public class Meetup {
+class Meetup {
     private final LocalDate startOfMonth;
 
     public Meetup(int monthOfYear, int year) {
```
Comment (FridaTveit) [inline exercises/meetup/.meta/src/reference/java/Meetup.java]: This can also be package private :)

Comment (c-thornton) [reply exercises/meetup/.meta/src/reference/java/Meetup.java]: Sorry for missing that one @FridaTveit, thought I got them all. Thanks for the feedback on this. I will get that one modified in a couple hours when I am in front of my computer again and modify the PR.

Comment (sjwarner-bp) [reply exercises/meetup/.meta/src/reference/java/Meetup.java]: Perfect!

**Comment by c-thornton** [comment]
Thanks @sjwarner-bp for the tip. I was wondering what I was doing incorrectly there. I should have read what @FridaTveit had written better. This is a learning experience for sure. The tests are running again now so hopefully it will pass this time and we should be good.

**Comment by c-thornton** [comment]
Doh! I should have known that honestly. Thanks for the link, I will read up on that tonight. I will make the correct change shortly and push it to my fork. Thanks for the guidance @sjwarner-bp and for being patient with me.

**Comment by sjwarner-bp** [comment]
That's no problem! Thanks for the speedy updates 👍

**Comment by c-thornton** [comment]
So one thing I just thought about @sjwarner-bp was consistency with the style. Since the Meetup method will not have a private declaration should I go ahead and remove it from the cycleToPrev method as well so that the style is consistent within the Meetup class?

**Comment by sjwarner-bp** [comment]
That's a good question! While changing the `private` methods to package-private wouldn't cause the build to fail, it is worth keeping in mind that we are trying to [minimise the accessibility of classes and members](http://jtechies.blogspot.com/2012/07/item-13-minimize-accessibility-of.html). 

Because these methods are only used in Meetup (i.e. nothing outside of their class calls them), we can keep them as private.

Remember - package-private is **more** accessible than `private`, and we want to keep accessibility to a (working) minimum 🙂

**Comment by c-thornton** [comment]
Awesome, thanks for the link and the info. So I removed private from cycleToPrev and cycleToNext with the commit that I just pushed on my side. I added them back in a pushed again and the checks just finished up for this one and were succesful. Thanks for the extra guidance and patience while working with me on this pull request.

**Comment by c-thornton** [comment]
Awesome, thanks for the link and the info. So I removed private from cycleToPrev and cycleToNext with the commit that I just pushed on my side. I added them back in a pushed again and the checks just finished up for this one and were succesful. Thanks for the extra guidance and patience while working with me on this pull request.

**Comment by c-thornton** [comment]
Awesome! Thanks @sjwarner-bp. Now time to do some reading before tackling some other issues. Thanks for all the feedback.

**Pull request merged**